cypress/support/pages/HomePage.js.orig

```diff
@@ -0,0 +1,10 @@
+class HomePage {
```

Comment:
Are these auto-generated? If so, we should add `*.orig` to .gitignore.

Reply:
*.orig was added to .gitignore